src/libnml/os_intf/inetnull.cc

```diff
@@ -55,7 +56,9 @@ char *inet_file_gets(char *str, int maxlen, INET_FILE * inet_file)
 int inet_file_close(INET_FILE * inet_file)
 {
     if (NULL != inet_file) {
-	fclose(inet_file->fp);
+	if (inet_file->fp) {
+	    fclose(inet_file->fp);
```

Comment:
I believe this code need to do inet_file->fp = NULL; too.

Reply:
A few lines down, just outside that patch, in line 62, the whole inet_file structure gets deleted.

Reply:
Done, also in the destructor (which felt a bit weird, admittedly), and rebased to master.